src/map/itemdb.hpp

```diff
@@ -2259,10 +2259,11 @@ struct item_data
 //		some script commands should be revised as well...
 	uint64 class_base[3];	//Specifies if the base can wear this item (split in 3 indexes per type: 1-1, 2-1, 2-2)
 	uint16 class_upper; //Specifies if the class-type can equip it (See e_item_job)
-	struct {
-		int chance;
-		int id;
-	} mob[MAX_SEARCH]; //Holds the mobs that have the highest drop rate for this item. [Skotlex]
+	struct drop_chance {
```

Comment:
Would be nice to move this outside and follow the naming standards.

Reply:
What's the naming standards?
This struct only has a meaning inside item_data, so i made it a struct inside.